src/main/java/edu/cornell/kfs/coa/businessobject/CuAccountGlobal.java

```diff
@@ -2,164 +2,903 @@
-                    account.setIncomeStreamFinancialCoaCode(incomeStreamFinancialCoaCode);
-                }
-    
-                // INCOME STREAM ACCOUNT NUMBER
-                if (StringUtils.isNotBlank(incomeStreamAccountNumber)) {
-                    account.setIncomeStreamAccountNumber(incomeStreamAccountNumber);
-                }
-    
-                // CG CATL FED DOMESTIC ASSIST NBR
-                if (StringUtils.isNotBlank(accountCfdaNumber)) {
-                    account.setAccountCfdaNumber(accountCfdaNumber);
-                }
-    
-                // FINANCIAL HIGHER ED FUNCTION CODE
-                if (StringUtils.isNotBlank(financialHigherEdFunctionCd)) {
-                    account.setFinancialHigherEdFunctionCd(financialHigherEdFunctionCd);
-                }
-    
-                // SUFFICIENT FUNDS CODE
-                if (StringUtils.isNotBlank(accountSufficientFundsCode)) {
-                    account.setAccountSufficientFundsCode(accountSufficientFundsCode);
-                }
+    protected String majorReportingCategoryCode;
+    protected String accountPhysicalCampusCode;
+    protected Date accountEffectiveDate;
+    protected Boolean accountOffCampusIndicator;
+    protected Boolean closed;
+    protected String accountTypeCode;
+    protected String appropriationAccountNumber;
+    protected Boolean accountsFringesBnftIndicator;
+    protected String reportsToChartOfAccountsCode;
+    protected String reportsToAccountNumber;
+    protected String accountRestrictedStatusCode;
+    protected Date accountRestrictedStatusDate;
+    protected String endowmentIncomeAcctFinCoaCd;
+    protected String endowmentIncomeAccountNumber;
+    protected String programCode;
+    protected String budgetRecordingLevelCode;
+    protected Boolean extrnlFinEncumSufficntFndIndicator;
+    protected Boolean intrnlFinEncumSufficntFndIndicator;
+    protected Boolean finPreencumSufficientFundIndicator;
+    protected Boolean financialObjectivePrsctrlIndicator;
+    protected String contractControlFinCoaCode;
+    protected String contractControlAccountNumber;
+    protected String acctIndirectCostRcvyTypeCd;
+    protected Integer contractsAndGrantsAccountResponsibilityId;
+    protected String invoiceFrequencyCode;
+    protected String invoiceTypeCode;
+    protected Long costShareForProjectNumber;
+    protected boolean removeAccountExpirationDate;
+    protected boolean removeContinuationChartAndAccount;
+    protected String financialIcrSeriesIdentifier;
+    protected Boolean everify;
+    protected boolean removeIncomeStreamChartAndAccount;  
     
-                // LABOR BENEFIT RATE CATEGORY CODE
-                if (StringUtils.isNotBlank(getLaborBenefitRateCategoryCode())) {
-                    account.setLaborBenefitRateCategoryCode(getLaborBenefitRateCategoryCode());
-                }
+	protected MajorReportingCategory majorReportingCategory;
```

Comment:
Small indentation issue on this line; not worth fixing unless other changes are needed.